draft-ietf-quic-transport.md

```diff
@@ -1525,6 +1519,36 @@ initial_max_uni_streams (0x0008):
   equivalent to receiving a MAX_STREAM_ID containing 18 when received by a
   client or 19 when received by a server.
 
+initial_max_stream_data_bidi_local (0x0000):
+
+: The initial stream maximum data for bidirectional, local streams parameter
+  contains the initial value for the maximum data that can be sent on any newly
+  created bidirectional stream opened by the endpoint that sets the transport
+  parameter.  This parameter is encoded as an unsigned 32-bit integer in units
+  of octets.  This is equivalent to a MAX_STREAM_DATA frame
+  ({{frame-max-stream-data}}) being sent on those streams immediately after
+  opening.  In client transport parameters, this applies to streams with an
+  identifier ending in 0x0; in server transport parameters, this applies to
+  streams ending in 0x1.  This parameter defaults to 0 if absent.
+
+initial_max_stream_data_bidi_remote (0x000a):
+
+: The initial stream maximum data for bidirectional, remote streams parameter is
+  equivalent to the initial_max_stream_data_bidi_local parameter, but it applies
+  instead to bidirectional streams initiated by a remote peer.  In client
+  transport parameters, this applies to streams with an identifier ending in
+  0x1; in server transport parameters, this applies to streams ending in 0x0.
+  Thisparameter defaults to 0 if absent.
+
+initial_max_stream_data_uni (0x000b):
+
+: The initial stream maximum data for bidirectional, remote streams parameter is
+  equivalent to the initial_max_stream_data_bidi_local parameter, but it applies
```

Comment:
Having these two be "equivalent to" the first one feels a little odd, since there's nothing particularly special about the first one over the others.  I'd be tempted to either repeat the verbiage in each, or pull it out to an unbulleted paragraph that introduces these three as a related group.